src/Demos/HSP.lagda

```diff
@@ -1754,6 +1754,8 @@ t~⟧}~\afld{⟨\$⟩}~\ab{ρ}\\[-8pt]
 %
 The \defn{relatively free algebra over} \ab{X} (relative to
 \ab{𝒦}) is defined to be the quotient \Free{X} := \T{X}~\af{/}~\afld{≈}.
+As we are using setoids, this can be done by changing the equivalence relation
```

Comment:
Looks good... though I might revise it to make it a bit more clear what "this" is.